src/components/GravatarProfile/GravatarProfile.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import md5 from 'md5';
+
+import './GravatarProfile.css';
+
+const GravatarProfile = ({email, gravStyle}) => (
+	<div className='GravatarProfile'>
+		<img src={'https://www.gravatar.com/avatar/' + md5(email.trim().toLowerCase()) + '?d=' + gravStyle} />
```

Comment:
alt attribute is required on the element (however it can just be an empty string)

Reply:
Specifically i think it _should_ be an empty string for accessibility reasons, right? Going off something @peasandwell mentioned

Reply:
noted. ive added an empty alt text